nodes/KlickTipp/actions/field/get.operaton.ts

```diff
@@ -0,0 +1,46 @@
+import type { IExecuteFunctions, INodeProperties } from 'n8n-workflow';
+import { apiRequest } from '../../transport';
+import { handleError, handleObjectResponse, updateDisplayOptions } from '../../utils/utilities';
+
+export const properties: INodeProperties[] = [
+  {
+    // eslint-disable-next-line n8n-nodes-base/node-param-display-name-wrong-for-dynamic-options
+    displayName: 'Field ID',
+    name: 'fieldId',
+    type: 'options',
+    typeOptions: {
+      loadOptionsMethod: 'getFields',
+    },
+    default: '',
+    options: [],
+    description:
+      'Choose from the list, or specify an ID using an <a href="https://docs.n8n.io/code-examples/expressions/">expression</a>',
+  },
+];
+
+const displayOptions = {
+  show: {
+    resource: ['field'],
+    operation: ['get'],
+  },
+};
+
+export const description = updateDisplayOptions(displayOptions, properties);
+
+export async function execute(this: IExecuteFunctions, index: number) {
+  const fieldId = this.getNodeParameter('fieldId', index) as string;
+
+  //Extract just field name, i.e. CompanyName
+  const fieldName = fieldId.replace(/^field/, '');
+
+  if (!fieldName) {
+    return handleError.call(this, 'The field ID is required.');
```

Comment:
nit: here the same: use field ID or name for both, would be more clear I suppose or we need to exit before already

Reply:
Fixed, I also duplicated these changes in Node-RED app